tests/runtimes/simple-keyvalue/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "simple-keyvalue-runtime"
```

```suggestion
name = "test-runtime-simple-keyvalue"
```
I think this would make it explicit that this is something only for tests.